src/node_jsvmapi.h

```diff
@@ -94,60 +81,42 @@ NODE_EXTERN napi_status napi_create_object(napi_env e, napi_value* result);
 NODE_EXTERN napi_status napi_create_array(napi_env e, napi_value* result);
 NODE_EXTERN napi_status napi_create_array_with_length(napi_env e, int length,
                                                       napi_value* result);
-NODE_EXTERN napi_status napi_create_number(napi_env e,
-                                           double val,
+NODE_EXTERN napi_status napi_create_number(napi_env e, double val,
                                            napi_value* result);
-NODE_EXTERN napi_status napi_create_string_utf8(napi_env e,
-                                                const char* s,
-                                                int length,
-                                                napi_value* result);
-NODE_EXTERN napi_status napi_create_string_utf16(napi_env e,
-                                                 const char16_t* s,
+NODE_EXTERN napi_status napi_create_string_utf8(napi_env e, const char* s,
+                                                int length, napi_value* result);
+NODE_EXTERN napi_status napi_create_string_utf16(napi_env e, const char16_t* s,
```

Comment:
For these cases, when I looked at the existing code (I checked about 15-20 files)  in core, when there were multiple lines for the parameters the examples I found had one line per parameter. That's why I used that format in the changes I made to this file.  I think we should stick to that here and in the other files.